numpy/_core/src/multiarray/array_api_standard.c

```diff
@@ -0,0 +1,78 @@
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+#define _MULTIARRAYMODULE
+
+#include <numpy/ndarraytypes.h>
+
+
+NPY_NO_EXPORT PyObject *
+array_device(PyObject *NPY_UNUSED(self), void *NPY_UNUSED(ignored))
+{
+    return PyUnicode_FromString("cpu");
+}
+
+NPY_NO_EXPORT PyObject *
+array_to_device(PyObject *self, PyObject *args, PyObject *kwds)
+{
+    static char *kwlist[] = {"", "stream", NULL};
+    char *device = "";
+    PyObject *stream = Py_None;
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "s|$O:to_device", kwlist,
+                                     &device,
+                                     &stream)) {
+        return NULL;
+    }
+
+    if (stream != Py_None) {
+        PyErr_SetString(PyExc_ValueError,
+                        "The stream argument in to_device() "
+                        "is not supported");
+        return NULL;
+    }
+
+    if (strcmp(device, "cpu") != 0) {
+        PyErr_Format(PyExc_ValueError,
+                     "Unsupported device: %s.", device);
```

Comment:
Maybe this instead?
```suggestion
                     "device kwarg must be 'cpu' not: %s.", device);
```

Reply:
I would still use "Unsupported" word here to indicate why the `"cpu"` is only accepted.

How about: `"Unsupported device: %s. Only 'cpu' is accepted."`?

Reply:
Done!